src/webpack.config.ts

```diff
@@ -237,6 +238,25 @@ function webpackConfig(args: Partial<BuildArgs>) {
 					return [
 						{ test: /custom-element\.js/, loader: `imports-loader?widgetFactory=${args.element}` }
 					];
+				}),
+				...includeWhen(args.bundles && Object.keys(args.bundles).length, () => {
+					const loaders: any[] = [];
+
+					Object.keys(args.bundles).forEach(bundleName => {
+						(args.bundles || {})[ bundleName ].forEach(fileName => {
```

Comment:
I've run into this problem with TS before. I could probably have put `args.bundles!` as well. It seems like the type inference for variables doesn't extend into closures, as TS seems to forget that `args.bundles` wasn't null a second ago :)

Removing the `|| {}` causes a "variable could be undefined" error. Any thoughts?